build-system/test-configs/forbidden-terms.js

```diff
@@ -727,20 +727,14 @@ const forbiddenTermsGlobal = {
       'extensions/amp-iframe/0.1/storybook/Basic.amp.js',
       'extensions/amp-iframe/1.0/storybook/Basic.amp.js',
       'extensions/amp-image-slider/0.1/storybook/Basic.amp.js',
-      'extensions/amp-inline-gallery/1.0/storybook/Basic.js',
-      'extensions/amp-lightbox/1.0/storybook/Basic.amp.js',
       'extensions/amp-lightbox/1.0/storybook/Basic.js',
-      'extensions/amp-lightbox-gallery/1.0/storybook/Basic.amp.js',
-      'extensions/amp-lightbox-gallery/1.0/storybook/Basic.js',
       'extensions/amp-selector/1.0/storybook/Basic.amp.js',
       'extensions/amp-selector/1.0/storybook/Basic.js',
       'extensions/amp-sidebar/0.1/storybook/Basic.amp.js',
       'extensions/amp-sidebar/1.0/storybook/Basic.amp.js',
       'extensions/amp-sidebar/1.0/storybook/Basic.js',
       'extensions/amp-soundcloud/1.0/storybook/Basic.amp.js',
       'extensions/amp-soundcloud/1.0/storybook/Basic.js',
-      'extensions/amp-stream-gallery/1.0/storybook/Basic.amp.js',
-      'extensions/amp-stream-gallery/1.0/storybook/Basic.js',
```

Comment:
We should be able to approve this file but I messed up the ownership. Please pull after merging #37751, and we should be able to merge this PR.

Reply:
I rebased and force pushed, but I'm still seeing a "missing owners approval" red flag